### ISSUE TITLE
APIs which required authentication are not showing with swagger-UIv3

### ISSUE THREAD

**Issue by x86kernel** [marcgibbons/django-rest-swagger#755]
Although after authorization, APIs which required authentication were not shown with swagger-UIv3
So, I downgrade swagger to 2.1.0 and it worked with same settings, same APIs

**Comment by koslibpro** [comment]
That's right, I was just about to open the exact same issue. Nice UI work guys btw :)

**Comment by marcgibbons** [comment]
Are you referring to the authenticate button? I may have accidentally removed it from the default settings; but you can always add it back in by configuring `SECURITY_DEFINTIONS`

I also noticed that in v3 the headers don't get through the SwagggerUI auth; fix is coming in next version release.

**Comment by cr17ms** [comment]
Same problem with JWT Authentication.

**Comment by marcoceccarellispotsoftware** [comment]
hi
it's something related to SwaggerUI v3 that has changed this beahaviour.

swagger-api/swagger-ui#4883
swagger-api/swagger-ui#4325
swagger-api/swagger-ui#4324

other details
axnsan12/drf-yasg#131
 
@marcgibbons concerning the headers refer to my comment on #762

**Comment by cegprakash** [comment]
adding another similar issue : https://github.com/marcgibbons/django-rest-swagger/issues/757 which talks about a fix for this

**Comment by amol1992-hub** [comment]
Hi

Did you guys get this issue resolved ?

**Comment by megazza** [comment]
Hi

Quite an old thread, but I still see this as an issue in 2.2.0.  Even with the api_key SECURITY_DEFINITIONS set in SWAGGER_SETTINGS, the AUTHORIZE (when using "Bearer <mytoken>" does not expose the protected resources in the UI.   2.1.2 works fine without any of these settings.
